scripts/generate-app-config/injectTestEvidenceLink.js

```diff
@@ -0,0 +1,29 @@
+/**
+* Return navigational links with the config for test evidence injected.
+*/
+const injectTestEvidenceLink = (navConfig) => {
+  const { navigation } = navConfig;
+  let addScreenshots = true;
+  if (navigation.links.length) {
+    for (let i = 0; i < navigation.links.length; i += 1) {
+      const navigationItem = navigation.links[i];
+      if (navigationItem.path === '/evidence') {
+        addScreenshots = false;
+        break;
```

Comment:
could we return navigation.links automatically here and then remove the addScreenshots boolean / if (addScreenshots) below?